recipe/build.sh

```diff
@@ -1,17 +1,21 @@
-#!/bin/bash
+#!/usr/bin/env bash
+set -eux
 
-mkdir build && cd build
+$PYTHON $RECIPE_DIR/builder.py
 
-cmake -LAH \
-  -DCMAKE_PREFIX_PATH="$PREFIX" \
-  -DCMAKE_INSTALL_PREFIX="$PREFIX" \
-  -DCMAKE_INSTALL_LIBDIR=lib \
-  -DCMAKE_BUILD_TYPE=Release  \
-  -DPYTHON_EXECUTABLE=${PREFIX}/bin/python \
-  -DJPEG_INCLUDE_DIR=${PREFIX}/include \
-  -DPNG_PNG_INCLUDE_DIR=${PREFIX}/include \
-  -DPNG_INCLUDE_DIR=${PREFIX}/include \
-  -Dsqlite_path=${PREFIX}/include \
-  ../tools/python
-make -j$CPU_COUNT
-cp dlib*.so $SP_DIR
+# previous script for posterity
```

Comment:
This is all in the git history so let's just remove it

Reply:
42d9523